### PR TITLE
[1LP][WIP] fix perf tests

### PR DESCRIPTION
**Pull request by otsuman**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 288a00441c4396bc19d6fa4cfc3514910343b57d:

`cfme/utils/smem_memory_monitor.py`:
- :bangbang: Line 853:33: **E271** *multiple spaces after keyword*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/smem_memory_monitor.py [hunk 1]
```diff
@@ -142,14 +141,15 @@
 
 
 class SmemMemoryMonitor(Thread):
-    def __init__(self, ssh_client, scenario_data):
+    def __init__(self, ssh_client, scenario_data, appliance):
```
Comment (psav) [inline cfme/utils/smem_memory_monitor.py]: **REQUIRED:** As we are refactoring this, let's remove the ssh_client argument entirely and just pass the appliance, then we can locally obtain ```ssh_client``` inside this method.

**Comment by dajoRH** [comment]
Lint report for commit 780ff2a8e1549ce09ff61ae1ccaf9c2f1c1ad051:

`cfme/utils/smem_memory_monitor.py`:
- :bangbang: Line 244:101: **E501** *line too long (106 > 100 characters)*
- :bangbang: Line 374:101: **E501** *line too long (108 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**